src/components/Dashboard.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useHistory } from 'react-router-dom';
+
+// App imports
+import { deleteItem, getData, postData } from '@utils/apiHooks';
+import { formatUIDate } from '@utils/date';
+import { USER_VOYAGE_REPORT_URL, VOYAGE_REPORT_URL } from '@constants/ApiConstants';
+import { EDIT_VOYAGE_CHECK_DETAILS_URL } from '@constants/ClientConstants';
+
+const Dashboard = (pageData) => {
+  const history = useHistory();
+  const [reportList, setReportList] = useState();
+
+  const getReportList = () => {
+    let validReports = [];
+    getData(USER_VOYAGE_REPORT_URL)
+      .then((resp) => {
+        if (!resp.errors) {
+          resp.items.map((report) => {
+            if (!report.departureDate && !report.departureTime && !report.departurePort) {
+              deleteItem(`${VOYAGE_REPORT_URL}/${report.id}`);
+            } else {
+              validReports.push(report);
+            }
+          });
+          setReportList(validReports);
+        }
+      });
+  };
+
+  const handleStart = () => {
+    postData(USER_VOYAGE_REPORT_URL, null, 'reports')
+      .then((resp) => {
+        history.push({
+          pathname: '/save-voyage/page-1',
+          state: { voyageId: resp.id },
+        });
+      });
```

Comment:
Using `async/await` could make this code slightly more readable:

```suggestion
    const resp = await postData(USER_VOYAGE_REPORT_URL, null, 'reports');
    history.push({
      pathname: SAVE_VOYAGE_DEPARTURE_URL,
      state: { voyageId: resp.id },
    });
  });
```

Reply:
Updated